### ISSUE TITLE
Passing a random argument to `LinkTo` yields a false positive on `no-unknown-arguments-for-builtin-components`

### ISSUE THREAD

**Issue by MikiDi** [ember-template-lint/ember-template-lint#2071]
Something like
```hbs
    <LinkTo
      class="auk-search-results-list__item"
      @route={{@route}}
      @models={{this.models}}
      @random="test"
      @query={{@query}}
      ...attributes
    >
```
yields
```
Passing the "@dragOver" argument to <LinkTo /> is deprecated.
Instead, please use the {{on}} modifier, i.e. "<LinkTo {{on "dragover" ...}} />" instead of "<LinkTo @dragOver={{...}} />".  no-unknown-arguments-for-builtin-components
```
Due to the fuzzy matching going on until it finds sómething https://github.com/ember-template-lint/ember-template-lint/blob/d50db604486b444065f8919e5484d25715f06d84/lib/rules/no-unknown-arguments-for-builtin-components.js#L297

**Comment by MikiDi** [comment]
If I'm not mistaken it is this https://github.com/ember-template-lint/ember-template-lint/pull/1823/files#diff-0d35c8339def5f04d4dd503125f81691f1c1fc72a726331bb90e29cdab3f6654R303 addition that makes it "return to soon" to reach the "unknown-argument"-part

**Comment by bertdeblock** [comment]
Maybe the fuzzy matching could be tweaked a bit using [some of these options](https://fusejs.io/api/options.html), though I'm guessing it will be difficult to make this bullet proof.

Maybe @lifeart has any ideas if this can be improved?

**Comment by lifeart** [comment]
As I remember Fuse has scoring, and we could filter results by score here

**Comment by lifeart** [comment]
But, likely we should emit "unknown argument" error first here, instead of saying that something is deprecated.

**Comment by lifeart** [comment]
Error should be:

`"@random" is unknown argument for <LinkTo /> component. Did you mean "@dragOver"?`;

**Comment by bertdeblock** [comment]
This can be closed since #2088 was merged and released in [v3.8.0](https://github.com/ember-template-lint/ember-template-lint/blob/master/CHANGELOG.md#v380-2021-10-01).
